inventory/rh-sso/hosts

```diff
@@ -9,3 +9,6 @@ rh-sso-hosts
 
 [rhsm_hosts:children]
 rh-sso-hosts
+
+[identity-hosts:children]
+rh-sso-hosts
```

Comment:
Since we're interacting with the API, can we just set this to `localhost` (and remove `children`)?

Reply:
Fixed.